pootle/static/js/editor/app.js

```diff
@@ -2159,7 +2159,8 @@ PTL.editor = {
 
     // FIXME: move this side-effect elsewhere
     if (results.length > 0 && results[0].source === sourceText) {
-      if (ReactEditor.stateValues[0] === '') {
+      const allowAutofill = this.settings.canSuggest || this.settings.canTranslate;
```

Comment:
`canSuggest` and `canTraslate` are taken from 
```
cansuggest=check_permission("suggest", request)
cantranslate=check_permission("translate", request)
```
It takes into account `superuser`. It's about possibility to edit value in the editor area only.
And I also realised that `allowAutofill` is flag which makes sence for JS-client only so I'd not calculate it in python.